internal/bam/bam.go

```diff
@@ -23,16 +23,14 @@ import (
 
 	"github.com/googlegenomics/htsget/internal/bgzf"
 	"github.com/googlegenomics/htsget/internal/binary"
+	"github.com/googlegenomics/htsget/internal/csi"
 	"github.com/googlegenomics/htsget/internal/genomics"
 )
 
 const (
 	baiMagic = "BAI\x01"
 	bamMagic = "BAM\x01"
```

Comment:
@anamanolache You are correct on the BAI magic number, but not correct on the BAM one.  Here's an example:

```Bash
$ od -c -N 4 sample-example.bam.bai
0000000   B   A   I 001
0000004
$ od -c -N 4 sample-example.bam
0000000 037 213  \b 004
0000004
```